### ISSUE TITLE
[Linux] Reading ID-card data fails if another smartcard (e.g YubiKey) is in slot 0

### ISSUE THREAD

**Issue by andres-liiver** [open-eid/DigiDoc4-Client#465]
The DigiDoc4 client fails to read ID-card data when there's another smartcard (e.g. YubiKey) in slot 0. However, it works if the ID-card is loaded first, in slots 0 and 1, before the YubiKey (slot 2).

Output of `pkcs11-tool -L`:
```
Available slots:
Slot 0 (0x0): Yubico YubiKey FIDO+CCID 00 00
  token label        : ***
  token manufacturer : piv_II
  token model        : PKCS#15 emulated
  token flags        : login required, rng, token initialized, PIN initialized
  hardware version   : 0.0
  firmware version   : 0.0
  serial num         : 00000000
  pin min/max        : 4/8
Slot 1 (0x4): Alcor Micro AU9560 01 00
  token label        : PIN1 (LIIVER,ANDRES,***)
  token manufacturer : AS Sertifitseerimiskeskus
  token model        : PKCS#15 emulated
  token flags        : login required, rng, token initialized, PIN initialized
  hardware version   : 0.0
  firmware version   : 0.0
  serial num         : ***
  pin min/max        : 4/12
Slot 2 (0x5): Alcor Micro AU9560 01 00
  token label        : PIN2 (LIIVER,ANDRES,***)
  token manufacturer : AS Sertifitseerimiskeskus
  token model        : PKCS#15 emulated
  token flags        : login required, rng, token initialized, PIN initialized
  hardware version   : 0.0
  firmware version   : 0.0
  serial num         : ***
  pin min/max        : 5/12
```

DigiDoc4 client shows a dropdown:

![image](https://user-images.githubusercontent.com/8243886/53945221-d1902600-40c9-11e9-9293-39f12cb93ccf.png)

Even with the ID-card selected, no data is read from the card:

![image](https://user-images.githubusercontent.com/8243886/53945327-0ac89600-40ca-11e9-9c32-42a3abcf915d.png)

Loading the picture works, however.

**Comment by Counter178** [comment]
It's known issue and in backlog. Thanks for the information!

**Comment by kristelmerilain** [comment]
Fixed by #422
